kafka-avro-schema-quickstart/docker-compose.yaml

```diff
@@ -0,0 +1,41 @@
+version: '2'
+
+services:
+
+  zookeeper:
+    image: strimzi/kafka:0.20.1-kafka-2.5.0
```

Comment:
That looks outdated.

Reply:
2.7.0 works well too, I missed it because I didn't check quay, updating